src/accounts/pages/Home.tsx

```diff
@@ -1,6 +1,315 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  Grid,
+  Typography,
+} from "@mui/material";
+import Share from "../assets/share.svg";
+import TImage from "../../common/atoms/TImage";
+import HyphaDAO from "../assets/hypha-dao.svg";
+import FiddleArt from "../assets/fiddle-art.svg";
+import { useUserStore } from "../../common/stores/user.store";
+import {
+  ExternalUser,
+  getLoginRequestFromUrl,
+  RequestsManager,
+  ResponsesManager,
+  SdkErrors,
+  terminateLoginRequest,
+} from "@tonomy/tonomy-id-sdk";
+import useErrorStore from "../../common/stores/errorStore";
+import LogoutIcon from "@mui/icons-material/Logout";
+
+import "./Home.css";
+import UserAvatar from "../assets/avatar.svg";
+
+import Debug from "debug";
 
 export default function Home() {
-  return <h1> {settings.config.appName} website</h1>;
+  const [username, setUsername] = useState<string>();
+  const { user, setUser, isLoggedIn, logout } = useUserStore();
+  const errorStore = useErrorStore();
+
+  useEffect(() => {
+    async function authentication() {
+      try {
+        const user = await ExternalUser.getUser({ autoLogout: false });
+        setUser(user);
+        const username = await user.getUsername();
+        if (!username) throw new Error("No username found");
+        setUsername(username.getBaseUsername());
+      } catch (e) {
+        console.log("e", e);
+      }
+    }
+    authentication();
+  }, []);
+
+  async function terminateLogin(error): Promise<string> {
+    const { requests } = await getLoginRequestFromUrl();
+    const managedRequests = new RequestsManager(requests);
+
+    const externalLoginRequest =
+      managedRequests.getLoginRequestWithDifferentOriginOrThrow();
+
+    const managedResponses = new ResponsesManager(managedRequests);
+
+    return (await terminateLoginRequest(managedResponses, "mobile", error, {
+      callbackOrigin: externalLoginRequest.getPayload().origin,
+      callbackPath: externalLoginRequest.getPayload().callbackPath,
+    })) as string;
+  }
+
+  const onLogout = async () => {
+    try {
+      const callbackUrl = await terminateLogin({
+        code: SdkErrors.UserLogout,
+        reason: "User logged out",
+      });
+
+      if (isLoggedIn()) await logout();
+
+      window.location.href = callbackUrl;
+    } catch (e) {
+      errorStore.setError({ error: e, expected: false });
+    }
+  };
+
+  const services = [
+    {
+      name: "Hypha DAO",
+      url: "hypha.earth",
+      description:
+        "A decentralized platform empowering communities to collaborate, govern, and grow together seamlessly",
+      logo: HyphaDAO,
+      icon: Share,
+    },
+    {
+      name: "Fiddle Art",
+      url: "tonomy.io",
+      description:
+        "A vibrant creative platform where artists showcase their work, connect with others, and collaborate globally",
+      logo: FiddleArt,
+      icon: Share,
+    },
+    {
+      name: "Demo website",
+      url: "demo.testnet.tonomy.io",
+      description:
```

Comment:
please write the correct description. or make up a better one. this is for the block explorer